hash/risc0/bench/run.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+if [ -z ${ZKBENCH_HASH_TYPE} ]; then 
+ZKBENCH_HASH_TYPE="sha256"    
+fi
+
+if [ -z ${ZKBENCH_WHICH} ]; then 
+ZKBENCH_WHICH="LINEAR"    
+fi
+
+if [ -z ${ZKBENCH_NTHREADS} ]; then 
+ZKBENCH_NTHREADS=1    
+fi
+
+if [ -z ${ZKBENCH_INPUT_SIZE_KB} ]; then 
+ZKBENCH_INPUT_SIZE_KB=1024
+fi
+
+if [ -z ${ZKBENCH_TREE_DEPTH} ]; then 
+ZKBENCH_TREE_DEPTH=4
+fi
+
+
+echo "Running benchmarks with the following configurations:"
+echo "HASH            = $ZKBENCH_HASH_TYPE"
+# echo "WHICH           = $ZKBENCH_WHICH"
+echo "NTHREADS        = $ZKBENCH_NTHREADS"
+echo "Input Size (KB) = $ZKBENCH_INPUT_SIZE_KB"
+
+# Run the benchmarks using cargo run
+# Check if the environment variable is set to "poseidon2_bn128"
+if [ "$ZKBENCH_HASH_TYPE" == "poseidon2_bn128" ]; then
+    # echo "Running Poseidon2 benchmark over bn128 field"
+    CARGO_BUILD_JOBS=$ZKBENCH_NTHREADS cargo run $ZKBENCH_HASH_TYPE $ZKBENCH_TREE_DEPTH
+else
+    CARGO_BUILD_JOBS=$ZKBENCH_NTHREADS cargo run $ZKBENCH_HASH_TYPE $ZKBENCH_INPUT_SIZE_KB
```

Comment:
on my machine `cargo run` does a recompilation all the time, which is first of all not good, second, it ruins our ability to measure the runtime. Can we build an executable in `build.sh` (I think we already do) and just run that executable here instead of `cargo run` here?

Reply:
`cargo run` always complies and then run the code. we can certainly use the executable from `cargo build`